<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frontEnd</class>
 <widget class="QMainWindow" name="frontEnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VT Motorsports Formula SAE Wireless Telemetry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="serialButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>310</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Available Serial (COM) Ports</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portsComboBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Telemetry</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
